tests/legacy-cli/e2e/tests/basic/e2e.ts

```diff
@@ -59,13 +59,6 @@ export default function () {
       // Should run side-by-side with `ng serve`
       .then(() => execAndWaitForOutputToMatch('ng', ['serve'], / Compiled successfully./))
       .then(() => ng('e2e', 'test-project', '--dev-server-target='))
-      // Should fail without updated webdriver
```

Comment:
Yeah, this is because the webdriver manager is already update when running https://github.com/angular/angular-cli/pull/23245/files#diff-6f3e609e1aead9e976e142745edcbcf23cbe30c3f61a400146fed645dfd751b2R47-R63

Reply:
Wasn't it before as well? The postinstall in the root package.json was running it and previously the tests were consuming that. Why is it different now that it's using the webdriver within the test project?

Reply:
That's being for E2E's the local package would be used as such the repository package was not being consumed.